site/src/Consumer/EmailNotificationConsumer.php

```diff
@@ -12,26 +13,82 @@
  */
 class EmailNotificationConsumer extends AbstractNotificationConsumer
 {
+    /**
+     * @var UserManagerInterface
+     */
+    protected $userManager;
+
+    /**
+     * @var \Swift_Mailer
+     */
+    protected $mailer;
+
+    /**
+     * @var Environment
+     */
+    protected $twig;
+
     /**
      * EmailNotificationConsumer constructor
      *
-     * @param LoggerInterface $logger
+     * @param LoggerInterface      $logger
+     * @param UserManagerInterface $userManager
+     * @param Environment          $twig
+     * @param \Swift_Mailer        $mailer
      */
-    public function __construct(LoggerInterface $logger)
+    public function __construct(
+        LoggerInterface $logger,
+        UserManagerInterface $userManager,
+        Environment $twig,
+        \Swift_Mailer $mailer
+    )
     {
         parent::__construct($logger);
+        $this->userManager = $userManager;
+        $this->twig = $twig;
+        $this->mailer = $mailer;
+    }
+
+    protected function shouldHandle()
+    {
+        $type = $this->notification->getType();
+        // TODO: Check subscription status also from database, after merge of #387
+        switch ($type) {
+            case "site_information":
+                $this->logger->info('Processing the notification and sending an email...');
+
+                return true;
+            default:
+                $this->logger->error('Something went wrong. Please check the notification type');
+
+                return false;
+        }
     }
 
     /**
      * Handles an email Notification
      *
      * @return bool
+     *
+     * @throws \Exception
      */
     protected function handle(): bool
     {
-        // TODO: Process the Notification($this->notification)
         echo "Email Notification Consumer";
 
+        $userEmail = $this->userManager->findUserByUsername($this->notification->getRecipient()->getUsername());
+
+        $message = (new \Swift_Message($this->notification->getSubject()))
+            // Set sender's email from a yaml file
+            ->setFrom('LIBRECORES_EMAIL')
```

Comment:
You need to make this email address configurable and read it from the settings yaml file.